crates/proto/src/rr/domain/name.rs

```diff
@@ -34,6 +34,9 @@ pub struct Name {
 }
 
 impl Name {
+    /// Maximum legal length of a domain name
+    pub const MAX_LENGTH: usize = u8::MAX as usize;
```

Comment:
I know it's the "same" us u8::MAX, but I think the RFC is specifies this as `255` explicitly.

Should we use that to be absolutely consistent with 1034?

```
To simplify implementations, the total number of octets that represent a
domain name (i.e., the sum of all label octets and label lengths) is
limited to 255.
```

Reply:
Sure, that makes sense. Will change